element.go

```diff
@@ -82,20 +258,13 @@ func (e *Element) OnMouseUp(h MouseEventHandler) {
 	e.onMouseEvent("mouseup", int(sjs.StopPropagation), h)
 }
 
-func (e *Element) ClassList() *TokenList {
-	return AsTokenList(e.v.Get("classList"))
-}
-
-func (e *Element) AttachShadow(open bool) *ShadowRoot {
+func (e *Element) AttachShadow(open bool, deligatesFocus bool) *ShadowRoot {
```

Comment:
Maybe instead of adding a second parameter (more in the future?) we can define an `AttachOptions` struct?

Reply:
Per documentation (https://developer.mozilla.org/en-US/docs/Web/API/Element/attachShadow) the `shadowRootInit` map supports only one field `mode`. But based on this https://github.com/w3c/webcomponents/issues/126 it has also `deligatesFocus`

`AttachOptions` is the best way to to account for future changes.